lib/screens/wrap_screen.dart

```diff
@@ -0,0 +1,48 @@
+import 'package:flutter/material.dart';
+import 'package:basic_widgets/strings.dart';
+
+class WrapScreen extends StatelessWidget {
+  const WrapScreen({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: const Text(Strings.wrap),
+      ),
+      body: Wrap(
+        spacing: 10.0,
+        runSpacing: 4.0,
+        children: <Widget>[
```

Comment:
You can avoid code duplication here using generator, something like this:
````
children: List<Widget>.generate(
              10,
              (i) => ElevatedButton(
                onPressed: () {},
                child: Text("Button $i"),
              ),
            ),
```

Reply:
add generate method to wrap screen